internal/internal_time.go

```diff
@@ -45,7 +45,7 @@ type (
 
 		// NewTimer - Creates a new timer that will fire callback after d(resolution is in seconds).
 		// The callback indicates the error(TimerCanceledError) if the timer is canceled.
-		NewTimer(d time.Duration, callback ResultHandler) *TimerID
+		NewTimer(d time.Duration, options TimerOptions, callback ResultHandler) *TimerID
```

Comment:
This may break PHP SDK since it's exposed via `internalbindings` but unsure if they use it. This seemed like the best way and should cause a clear compile break.

Reply:
I think they do but it should be fine